integration/appaccess/appaccess_test.go

```diff
@@ -600,16 +600,16 @@ func testAuditEvents(p *Pack, t *testing.T) {
 			},
 			PublicAddr: p.rootAppPublicAddr,
 		}
-		require.Empty(t, cmp.Diff(
+		return len(cmp.Diff(
```

Comment:
> This change fetches last 5 events for comparison and increases the wait time.

You also removed the `require.Empty()` call against the main `t`. This call was completely bypassing the `Eventually` from `ensureAuditEvent()` and failing the whole test immediately If the event was not received during the first run.

It was somewhat working before because `Eventually` was waiting at least one polling interval before checking the condition. Since https://github.com/stretchr/testify/pull/1427 this is not true anymore and `Eventually` immediately checks if the condition is true, triggering the bug described above.

I think this PR completely fixes the flaky test without even the timeout increase.